darts/models/__init__.py

```diff
@@ -3,15 +3,37 @@
 ------
 """
 
+from ..logging import get_logger
+
+logger = get_logger(__name__)
+
 # Forecasting
-from .arima import ARIMA, AutoARIMA
+try:
+    from arima import AutoARIMA
+except ModuleNotFoundError:
+    logger.warning("Support pmdarima based models not available. To enable it install darts[pmdarima]")
+
+try:
+    from .prophet import Prophet
+except ModuleNotFoundError:
+    logger.warning("Support Prophet based models not available. To enable it install darts[prophet]")
+
+try:
+    from .rnn_model import RNNModel
+except ModuleNotFoundError:
+    logger.warning("Support Torch based models not available. To enable it install darts[torch]")
+
+try:
+    from .tcn_model import TCNModel
+except ModuleNotFoundError:
+    logger.warning("Support Torch based models not available. To enable it install darts[torch]")
+
+
+from .arima import ARIMA
 from .baselines import NaiveMean, NaiveSeasonal, NaiveDrift
-from .prophet import Prophet
 from .exponential_smoothing import ExponentialSmoothing
-from .rnn_model import RNNModel
-from .tcn_model import TCNModel
-from .theta import Theta, FourTheta
 from .fft import FFT
+from .theta import Theta
```

Comment:
Any reason for removing `FourTheta` from the imports?

Reply:
I missed that one I'll put it back.